### DISCUSSION TITLE
Do we have any way to set something like /api/v1 in the routers?

### DISCUSSION THREAD

**Discussion by hoangtranson** [oakserver/oak#264] [General]
Presently, I am working on Deno project and I have to add `/api/v1` on every single route. 

Just don't know if we can have something like this.

```
app.use('/api/v1', router.routes());
```

In the https://github.com/oakserver/oak/blob/main/router.ts, we allow to set prefix like:

```
const router: Router = new Router({ prefix: "/api/v1"});
```

But this is kinda weird for me :D

**Comment by Srabutdotcom** [comment]
Hi @hoangtranson ,

Your question gave me an idea on how to define specific route in oak router.
I think we have to follow oak convention instead of building the variance that would make new confusion.

regards
